x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-studio.adoc

```diff
@@ -0,0 +1,293 @@
+= Use Anypoint Studio to Configure X12 EDI Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<conigure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log to check for problems, as described in <<view-app-log,View the App Log>>.
+
+You must change the default Anypoint configuration to use the X12 EDI connector 2.x.
+
+NOTE: If Anypoint Studio 7.1.0 and later requires more than the standard memory
```

Comment:
This is really not a note and doesn't belong here. It could be an additional configuration topic.